package.json

```diff
@@ -198,7 +198,6 @@
     "shelljs": "0.8.5",
     "sinon": "7.3.2",
     "snap-shot-it": "7.9.3",
-    "start-server-and-test": "1.10.8",
```

Comment:
`start-server-and-test` version `1` depends upon `tough-cookie` `2.5` but I don't see any use of `start-server-and-test` in the repo. I think we can remove this